plugins/modules/vmware_host_iscsi.py

```diff
@@ -687,6 +702,17 @@ def execute(self):
                 except Exception as e:
                     self.module.fail_json(msg="Failed to disable iSCSI: %s" % to_native(e))
 
+        if self.state == 'rescanned':
+            self.diff_iscsi_config()
+            if self.module.check_mode:
+                self.module.exit_json(changed=self.change_flag)
+
+            try:
+                self.host_obj.configManager.storageSystem.RescanHba(self.vmhba_name)
```

Comment:
There is module `vmware_host_scanhba` which does the same operation. Do you think we should rescan HBA by name support?

Reply:
Thank you @Akasurde for reviewing.

> There is module vmware_host_scanhba which does the same operation. Do you think we should rescan HBA by name support?

Oh, sorry, I have missed that the module.  
I confirmed that vmware_host_scanhba module also hasn't a problem working about what I wanted to do.  
So, I think that resscanned option isn't necessary this module.

I'll close this PR.